tools/c7n_azure/c7n_azure/storage_utils.py

```diff
@@ -47,6 +47,13 @@ def get_queue_client_by_uri(queue_uri, session=None):
 
         return queue_service, storage.container_name
 
+    @staticmethod
+    def create_queue_from_storage_account(storage_account, name):
+        keys = StorageUtilities.get_storage_keys(storage_account.id)
+        queue_service = QueueService(account_name=storage_account.name,
+                                     account_key=keys[0].value)
+        return queue_service.create_queue(name)
```

Comment:
If it fails, I think we need to add retry.

 I have seen issues of when I delete SA items like blobs and containers and when I try to recreate them again I get failure calls (maybe a 409).

Reply:
Per offline - added try-catch-log-exit instead.